api/oc_main.c

```diff
@@ -271,6 +272,23 @@ main_init_resources(void)
 #endif /* OC_SOFTWARE_UPDATE */
 }
 
+#ifdef OC_SECURITY
+static void
+main_set_resource_defaults(size_t device)
```

Comment:
Name `main_set_resource_defaults` doesn't seem right, something like `main_clear_storage`

Reply:
yeah, that's fine